content/common/contributing-docs.md

```diff
@@ -233,15 +238,63 @@ npx markdownlint-cli2 content/spin/install.md \"#node_modules\"
 
 ### 6.2 Generating Indexing For Your Content
 
-The documentation site implements in-house search and therefore it is recommended to generate a new index each time you update and push changes to the developer documentation repository. This is done simply by using the following command; in the developer directory:
+The developer documentation site implements in-house search and therefore it is recommended to generate a new index each time you update and push changes to the developer documentation repository. This is done simply by using the following command; in the developer directory:
 
 <!-- @selectiveCpy -->
 
 ```bash
 $ npm run build-index
 ```
 
-### 6.3 How To Properly Edit CSS Styles
+### 6.3 Increasing Search Visibility For Your Content
+
+The in-house search functionality is based on the indexing of individual words in each markdown file, which works well most of the time. However, there are a couple of scenarios where you _may_ want to deliberately increase search visibility for your content.
```

Comment:
I found the phrase "in-house" increasingly distracting as I encountered it more and more.  It seems to go to great lengths to emphasise that this is search functionality we built ourselves, and I'm not sure that's a useful part of the message.  I wonder if "built-in" or "integrated" would be a better fit?

Reply:
Went with "built-in" sounds great; thanks.